src/infra/main/template.yaml

```diff
@@ -357,6 +382,17 @@ Resources:
       LogGroupName: !Ref InterventionsProcessorFunctionLogGroup
       FilterPattern: "{$.message != \"Sensitive info*\"}"
 
+  InterventionsProcessorLambdaErrorMetricFilter:
+    Type: AWS::Logs::MetricFilter
+    Properties:
+      FilterPattern: '{ $.message = "Error caught, message will be retried." }'
```

Comment:
Hi @liamquayle , @chiarapascucci-gds , @amd-gds , @KG700  - could we add a unit test (to this PR) for the Intervention processor to check it logs this exact message so that this metric filter never gets broken through a code change.

Reply:
I'll push one up soon with a comment to explain why we need it